File: Python code - chapter 2/chapter2/example_7.py
# Example 7. A random variable takes on the value
# X = 1 if the outcome of rolling a die is strictly greater than 4
# and X= 0 otherwise.
# What is the probability distribution for X?
# Calculate E[X], Var(X).

from scipy.stats import bernoulli

if __name__ == '__main__':
    # X is a Bernoulli random variable with p=1/3.
    p = 1.0/3
    mean, var = bernoulli.stats(p, moments='mv')
    print(f'E[X] = %.4f' % mean)
    print(f'Var(X) = %.4f' % var)

File: R code - Chapter 2/example7.R
# Example 7. A random variable takes on the value X = 1 
# if the outcome of rolling a die is strictly greater than 4 
# and X= 0 otherwise. What is the probability distribution for X? 
# Calculate E[X], Var(X).
#
# Bernoulli(p)
p <- 1./3
cat("E[X] = ", p, "\n")
cat("Var(X) = ", p*(1-p), "\n")
